pre_commit/languages/rust.py

```diff
@@ -130,6 +141,12 @@ def install_environment(
     cli_deps = {
         dep for dep in additional_dependencies if dep.startswith('cli:')
     }
+    # Features starts with "--feature="
+    features = {
+        dep.replace(FEATURE_PREFIX, '')
+        for dep in additional_dependencies
+        if dep.startswith(FEATURE_PREFIX)
+    }
```

Comment:
I think we can probably just filter out anything that starts with `--` or doesn't have `:`?